unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/GLTF/AssetPromise_GLTF.cs

```diff
@@ -80,7 +80,14 @@ protected override void OnLoad(Action OnSuccess, Action<Exception> OnFail)
             {
                 gltfComponent = asset.container.AddComponent<GLTFComponent>();
 
-                gltfComponent.Initialize(webRequestController, AssetPromiseKeeper_GLTF.i.throttlingCounter);
+                IThrottlingCounter gltfThrottlingCounter = AssetPromiseKeeper_GLTF.i.throttlingCounter;
+
+                if (!DataStore.i.performance.throttleGLTF.Get())
```

Comment:
can we pass the correct `IThrottlingCounter` instance from the constructor? you can also create a proxy if a lazy implementation is needed

Reply:
Actually, this throttle was added as an experiment and forgot to remove it